src/NuGet.Core/NuGet.Protocol.Core.v3/LegacyFeed/PackageSearchResourceV2FeedProvider.cs

```diff
@@ -5,7 +5,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using NuGet.Protocol.Core.Types;
-using NuGet.Protocol.Core.v3;
+using NuGet.Protocol;
```

Comment:
Ah, yes. This was a big find and replace in preparation for merging this all into one assembly.